package.json

```diff
@@ -7,12 +7,12 @@
     "babel-polyfill": "6.6.1",
     "babel-runtime": "6.6.1",
     "bluebird": "3.3.3",
-    "body-parser": "1.15.0",
+    "body-parser": "^1.18.3",
     "classnames": "2.2.3",
     "cookie-parser": "1.4.1",
     "dispatchr": "^1.0.0",
     "eventemitter3": "^1.2.0",
-    "express": "4.13.4",
+    "express": "^4.16.3",
```

Comment:
are you updating these dependencies on purpose?

Reply:
fixed